nameparser/config/suffixes.py

```diff
@@ -25,6 +26,7 @@
 """
 SUFFIX_ACRONYMS = set([
     '(ret)',
+    '(ret.)',
```

Comment:
The "(ret)" was there when I started coding.  I didn't know how you might want to parse the "ret" and "vet" suffixes, so I tried to keep the parentheses in the parsed result.  I think I added trailing-period versions to the list before I realized that any parenthesis-delimited string would be parsed by the nickname routine.  I can certainly change how I'm processing these two as well as add other suffix strings to the preprocessing pattern.  Just let me know.

Reply:
What do you want to happen with ret and vet?

Reply:
Does something fail if you remove the one with the period?

Reply:
No.  I'm using a regex pattern for the **(ret)** and **(vet)** suffixes.  The regex pattern ignores any trailing period.  Using the existing list of suffixes doesn't work, since the nickname processing picks these up, matching the **parenthesis** pattern.  Nickname processing happens before suffix processing, which is why I added a suffix preprocessor routine for these two parenthesis-delimited suffixes.

I can retain the parentheses or drop them.  Let me know and I'll make sure the code does what you want, relative to these two suffixes.

Reply:
If you have changed the way that suffixes are detected then I think you need to modify the `is_suffix` method to implement that test using your new regexes.